frontend/src/metabase/nav/components/search/RecentsList/RecentsList.tsx

```diff
@@ -0,0 +1,176 @@
+import { useMemo } from "react";
+import { push } from "react-router-redux";
+import { t } from "ttag";
+import type { IconName } from "metabase/core/components/Icon";
+import { getTranslatedEntityName } from "metabase/common/utils/model-names";
+import { useListKeyboardNavigation } from "metabase/hooks/use-list-keyboard-navigation";
+import * as Urls from "metabase/lib/urls";
+import { SearchLoadingSpinner } from "metabase/nav/components/search/SearchResults/SearchResults";
+import { SearchResultLink } from "metabase/search/components/SearchResultLink";
+import type { RecentItem, UnrestrictedLinkEntity } from "metabase-types/api";
+import { useRecentItemListQuery } from "metabase/common/hooks";
+import RecentItems from "metabase/entities/recent-items";
+import { useDispatch } from "metabase/lib/redux";
+import { isSyncCompleted } from "metabase/lib/syncing";
+import {
+  LoadingSection,
+  ModerationIcon,
+  ResultNameSection,
+  ResultTitle,
+  SearchResultContainer,
+  ItemIcon,
+} from "metabase/search/components/SearchResult";
+import { Group, Loader, Stack, Title, Paper } from "metabase/ui";
+import EmptyState from "metabase/components/EmptyState";
+import { EmptyStateContainer } from "../SearchResults/SearchResults.styled";
```

Comment:
`SearchLoadingSpinner` is imported with absolute path, but `EmptyStateContainer` is imported with a relative path.